src/azul/service/manifest_service.py

```diff
@@ -608,7 +613,8 @@ def entity_type(self) -> str:
     def source_filter(self) -> SourceFilters:
         return [
             *super().source_filter,
-            'contents.files.related_files'
+            'contents.files.related_files',
+            'contents.files.drs_path'
```

Comment:
Redundant, if not harmful, since the super method already does that automatically.